lib/clients/backbeat-2017-07-01.api.json

```diff
@@ -38,6 +38,10 @@
                         "locationName": "Content-Length",
                         "type": "long"
                     },
+                    "PartNumber": {
+                        "location": "header",
```

Comment:
as discussed offline, please provide the location header name starting with `X-Scal-` such as `X-Scal-Part-Number`.